public/js/utils/source-map.js

```diff
@@ -5,9 +5,17 @@ import type { Location } from "./actions/types";
 const { getSource, getSourceByURL } = require("../selectors");
 const { isEnabled, getValue } = require("../feature");
 
-const sourceMapWorker = new Worker(
-  getValue("baseWorkerURL") + "source-map-worker.js"
-);
+let sourceMapWorker;
+function restartWorker() {
+  if(sourceMapWorker) {
+    sourceMapWorker.terminate();
+  }
+  sourceMapWorker = new Worker(
+    getValue("baseWorkerURL") + "source-map-worker.js"
+  );
+}
+restartWorker();
```

Comment:
minor thing - but `restartWorker` should probably be in the same util as `workerTask`